### PR TITLE
Added possibility to change Player buttons with Emojis ⚡

### PR DESCRIPTION
**Pull request by Nickeldon**

### CODE REVIEW

File: events/Process/process_tools.js
Comment (revot334) [inline events/Process/process_tools.js:1]: i would also prefer not use the chalk, please use default js console.log, console.error, ect

File: events/Process/process_tools.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      );
+      output = translatedArray.join(" ");
+    }
+
+    return output;
+  },
+
+  GetTranslationModule: async () => {
+    try {
+      const module = await import("translate");
```
Comment (revot334) [inline events/Process/process_tools.js:73]: use require keep it like the rest of the code space

File: events/Process/process_tools.js
Comment (revot334) [reply events/Process/process_tools.js:1]: is there a reason ? you prefer it ik it makes it look nice but I would prefer to keep it simple

File: package.json [hunk 2]
```diff
@@ -10,8 +10,10 @@
     },
     "dependencies": {
         "@discord-player/extractor": "^4.3.1",
-        "@discordjs/opus": "^0.9.0",
         "@distube/ytdl-core": "^4.11.8",
+        "@evan/opus": "^1.0.3",
+        "@discordjs/opus": "^0.9.0",
+        "chalk": "^5.3.0",
```
Comment (revot334) [inline package.json:16]: read above comments

**Review by revot334 — CHANGES_REQUESTED** [review]
please read code review a few things to fix!

File: events/Process/process_tools.js (hunk 1 above)
Comment (Nickeldon) [reply events/Process/process_tools.js:73]: It is an ES module

File: events/Process/process_tools.js
Comment (Nickeldon) [reply events/Process/process_tools.js:1]: done 👍

[the remark this answers is not in the document]
Comment (Nickeldon) [reply events/Process/process_tools.js:1]: Fixed 💀

File: main.js [hunk 3]
```diff
@@ -17,6 +17,13 @@ client.config = require('./config');
 const player = new Player(client, client.config.opt.discordPlayer);
 player.extractors.loadDefault();
 
+console.clear()
 require('./loader');
 
-client.login(client.config.app.token);
\ No newline at end of file
+client.login(client.config.app.token)
+.catch(async (e) => {
```
[the remark this answers is not in the document]
Comment (Nickeldon) [reply main.js:24]: Fixed 👍

**Review by revot334 — APPROVED** [review]
looks good!

**Pull request merged**